src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ShowPlan/Attributes.cs

```diff
@@ -283,5 +283,32 @@ public bool LongString
         private bool longString = false;
     }
 
+    public sealed class BetterValueAttribute: Attribute
+    {
+        public BetterValueAttribute()
+        {
+            this.value = BetterValue.None;
+        }
+
+        public BetterValueAttribute(BetterValue value){
+            this.value = value;
+        }
+
+        public BetterValue Value
+        {
+            get { return this.value; }
+        }
+
+        private BetterValue value;
+    }
+
+    public enum BetterValue
+    {
+        LowerNumber = 0,
+        HigherNumber = 1,
+        True = 2,
+        False = 3,
```

Comment:
`BetterValue.False` isn't being used at the moment. What's the need for it?

Reply:
If in case a false value is better than the true value. Even though it is not used here some other provider might have use for It in the future.